assets/src/components/form/layouts/two-column-admin/form-column.js

```diff
@@ -31,8 +31,8 @@ const FormColumn = ( {
 	}
 	htmlClass = htmlClass ? `${ htmlClass } ` : '';
 	htmlClass = offset ?
-		`${ htmlClass }col-sm-${ colSize } offset-sm-${ offset } ee-form-col` :
-		`${ htmlClass }col-sm-${ colSize } ee-form-col`;
+		`${ htmlClass } col-sm-${ colSize } offset-sm-${ offset } ee-form-col` :
+		`${ htmlClass } col-sm-${ colSize } ee-form-col`;
```

Comment:
this started off utilizing the styles that are used by `react-rrule-generator`.
I've contemplated changing all of these class names so that they are specific to EE to avoid any potential conflicts, but would have to duplicate a whole bunch of CSS.